code/frameworks/svelte-webpack5/package.json

```diff
@@ -64,7 +64,7 @@
   },
   "peerDependencies": {
     "storybook": "workspace:^",
-    "svelte": "^4.0.0 || ^5.0.0",
+    "svelte": "^5.0.0",
```

Comment:
We're dropping support for Svelte + Webpack in v9, so we should remove this whole package I guess.